### PR TITLE
Update Postfixadmin to 3.2.2

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: library/postfixadmin [hunk 1]
```diff
@@ -1,18 +1,18 @@
-# This file is generated via https://github.com/postfixadmin/docker/blob/a6b5fe2423a7e1a805d1fcff910049768c638cff/generate-stackbrew-library.sh
+# This file is generated via https://github.com/postfixadmin/docker/blob/8205c9e5eff6190c76edf6e59f8c28eb2b0306b2/generate-stackbrew-library.sh
 Maintainers: David Goodwin <david@codepoets.co.uk> (@DavidGoodwin)
 GitRepo: https://github.com/postfixadmin/docker.git
 
-Tags: 3.2-apache, 3-apache, apache, 3.2, 3, latest
-Architectures: amd64, arm64v8, i386
+Tags: 3.2.2-apache, 3.2-apache, 3.2.2, 3.2
```
Comment (yosifkit) [inline library/postfixadmin]: Did you mean to remove the `3` and `latest` tags? (and `fpm` and `fpm-alpine` below)

Comment (J0WI) [reply library/postfixadmin]: Oops, thanks for the hint!

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 7ab9222..dfc8ae2 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,9 +1,18 @@
 postfixadmin:fpm @ amd64
+postfixadmin:fpm @ arm32v5
+postfixadmin:fpm @ arm32v7
 postfixadmin:fpm @ arm64v8
 postfixadmin:fpm @ i386
+postfixadmin:fpm @ ppc64le
 postfixadmin:fpm-alpine @ amd64
+postfixadmin:fpm-alpine @ arm32v6
+postfixadmin:fpm-alpine @ arm32v7
 postfixadmin:fpm-alpine @ arm64v8
 postfixadmin:fpm-alpine @ i386
+postfixadmin:fpm-alpine @ ppc64le
 postfixadmin:latest @ amd64
+postfixadmin:latest @ arm32v5
+postfixadmin:latest @ arm32v7
 postfixadmin:latest @ arm64v8
 postfixadmin:latest @ i386
+postfixadmin:latest @ ppc64le
diff --git a/_bashbrew-list b/_bashbrew-list
index 35fb606..b498426 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -6,6 +6,10 @@ postfixadmin:3.2
 postfixadmin:3.2-apache
 postfixadmin:3.2-fpm
 postfixadmin:3.2-fpm-alpine
+postfixadmin:3.2.2
+postfixadmin:3.2.2-apache
+postfixadmin:3.2.2-fpm
+postfixadmin:3.2.2-fpm-alpine
 postfixadmin:apache
 postfixadmin:fpm
 postfixadmin:fpm-alpine
diff --git a/postfixadmin_fpm-alpine/Dockerfile b/postfixadmin_fpm-alpine/Dockerfile
index 50b565f..b1f2d6d 100644
--- a/postfixadmin_fpm-alpine/Dockerfile
+++ b/postfixadmin_fpm-alpine/Dockerfile
@@ -1,4 +1,4 @@
-FROM php:7.2-fpm-alpine
+FROM php:7.3-fpm-alpine
 LABEL maintainer="David Goodwin <david@codepoets.co.uk> (@DavidGoodwin)"
 
 # docker-entrypoint.sh dependencies
@@ -29,8 +29,8 @@ RUN set -ex; \
 	apk add --virtual .postfixadmin-phpexts-rundeps $runDeps; \
 	apk del .build-deps
 
-ARG POSTFIXADMIN_VERSION=3.2
-ARG POSTFIXADMIN_SHA512=9e022e3311bce96342e87925159a048f294ed0e2d4558bd43120b5c6eda35a7a2d128987372d310dfe86680c98a4d8cf3d602afcb20d948c9bba58bed1766560
+ARG POSTFIXADMIN_VERSION=3.2.2
+ARG POSTFIXADMIN_SHA512=6c84cb215e69c52c26db0651e5d0d9d8bcb0a63b00d3c197f10fa1f0442a1fde44bb514fb476a1e68a21741d603febac67282961d01270e5969ee13d145121ee
 
 ENV POSTFIXADMIN_VERSION $POSTFIXADMIN_VERSION
 ENV POSTFIXADMIN_SHA512 $POSTFIXADMIN_SHA512
diff --git a/postfixadmin_fpm-alpine/docker-entrypoint.sh b/postfixadmin_fpm-alpine/docker-entrypoint.sh
index b8e1f35..b925c5b 100755
--- a/postfixadmin_fpm-alpine/docker-entrypoint.sh
+++ b/postfixadmin_fpm-alpine/docker-entrypoint.sh
@@ -5,9 +5,18 @@ POSTFIXADMIN_DB_TYPE=${POSTFIXADMIN_DB_TYPE:=sqlite}
 POSTFIXADMIN_DB_HOST=${POSTFIXADMIN_DB_HOST:=""}
 POSTFIXADMIN_DB_USER=${POSTFIXADMIN_DB_USER:=""}
 POSTFIXADMIN_DB_PASSWORD=${POSTFIXADMIN_DB_PASSWORD:=""}
-POSTFIXADMIN_SETUP_PASSWORD=${POSTFIXADMIN_SETUP_PASSWORD:=""}
+
+# topsecret99
+DEFAULT_SETUP_PASSWORD="791eb4ead7fd996c01bed30707ae27dd:b7910d09773104bf84c4f4951205d2198c7cfc4f"
+POSTFIXADMIN_SETUP_PASSWORD=${POSTFIXADMIN_SETUP_PASSWORD:=$DEFAULT_SETUP_PASSWORD}
+
 
 if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
+
+	if [ "${POSTFIXADMIN_SETUP_PASSWORD}" = "${DEFAULT_SETUP_PASSWORD}" ]; then
+		echo >&2 "WARNING: Using default setup.php password of : topsecret99"
+	fi
+
 	if ! [ -e index.php -a -e scripts/postfixadmin-cli.php ]; then
 		echo >&2 "Postfixadmin not found in $PWD - copying now..."
 		if [ "$(ls -A)" ]; then
@@ -58,6 +67,7 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 		echo "<?php
 		\$CONF['database_type'] = '${POSTFIXADMIN_DB_TYPE}';
 		\$CONF['database_host'] = '${POSTFIXADMIN_DB_HOST}';
+		\$CONF['database_port'] = '${POSTFIXADMIN_DB_PORT}';
 		\$CONF['database_user'] = '${POSTFIXADMIN_DB_USER}';
 		\$CONF['database_password'] = '${POSTFIXADMIN_DB_PASSWORD}';
 		\$CONF['database_name'] = '${POSTFIXADMIN_DB_NAME}';
diff --git a/postfixadmin_fpm/Dockerfile b/postfixadmin_fpm/Dockerfile
index 97eba53..8a5bb7f 100644
--- a/postfixadmin_fpm/Dockerfile
+++ b/postfixadmin_fpm/Dockerfile
@@ -1,4 +1,4 @@
-FROM php:7.2-fpm
+FROM php:7.3-fpm
 LABEL maintainer="David Goodwin <david@codepoets.co.uk> (@DavidGoodwin)"
 
 # Install required PHP extensions
@@ -34,8 +34,8 @@ RUN set -ex; \
 	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
 	rm -rf /var/lib/apt/lists/*
 
-ARG POSTFIXADMIN_VERSION=3.2
-ARG POSTFIXADMIN_SHA512=9e022e3311bce96342e87925159a048f294ed0e2d4558bd43120b5c6eda35a7a2d128987372d310dfe86680c98a4d8cf3d602afcb20d948c9bba58bed1766560
+ARG POSTFIXADMIN_VERSION=3.2.2
+ARG POSTFIXADMIN_SHA512=6c84cb215e69c52c26db0651e5d0d9d8bcb0a63b00d3c197f10fa1f0442a1fde44bb514fb476a1e68a21741d603febac67282961d01270e5969ee13d145121ee
 
 ENV POSTFIXADMIN_VERSION $POSTFIXADMIN_VERSION
 ENV POSTFIXADMIN_SHA512 $POSTFIXADMIN_SHA512
diff --git a/postfixadmin_fpm/docker-entrypoint.sh b/postfixadmin_fpm/docker-entrypoint.sh
index b8e1f35..b925c5b 100755
--- a/postfixadmin_fpm/docker-entrypoint.sh
+++ b/postfixadmin_fpm/docker-entrypoint.sh
@@ -5,9 +5,18 @@ POSTFIXADMIN_DB_TYPE=${POSTFIXADMIN_DB_TYPE:=sqlite}
 POSTFIXADMIN_DB_HOST=${POSTFIXADMIN_DB_HOST:=""}
 POSTFIXADMIN_DB_USER=${POSTFIXADMIN_DB_USER:=""}
 POSTFIXADMIN_DB_PASSWORD=${POSTFIXADMIN_DB_PASSWORD:=""}
-POSTFIXADMIN_SETUP_PASSWORD=${POSTFIXADMIN_SETUP_PASSWORD:=""}
+
+# topsecret99
+DEFAULT_SETUP_PASSWORD="791eb4ead7fd996c01bed30707ae27dd:b7910d09773104bf84c4f4951205d2198c7cfc4f"
+POSTFIXADMIN_SETUP_PASSWORD=${POSTFIXADMIN_SETUP_PASSWORD:=$DEFAULT_SETUP_PASSWORD}
+
 
 if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
+
+	if [ "${POSTFIXADMIN_SETUP_PASSWORD}" = "${DEFAULT_SETUP_PASSWORD}" ]; then
+		echo >&2 "WARNING: Using default setup.php password of : topsecret99"
+	fi
+
 	if ! [ -e index.php -a -e scripts/postfixadmin-cli.php ]; then
 		echo >&2 "Postfixadmin not found in $PWD - copying now..."
 		if [ "$(ls -A)" ]; then
@@ -58,6 +67,7 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 		echo "<?php
 		\$CONF['database_type'] = '${POSTFIXADMIN_DB_TYPE}';
 		\$CONF['database_host'] = '${POSTFIXADMIN_DB_HOST}';
+		\$CONF['database_port'] = '${POSTFIXADMIN_DB_PORT}';
 		\$CONF['database_user'] = '${POSTFIXADMIN_DB_USER}';
 		\$CONF['database_password'] = '${POSTFIXADMIN_DB_PASSWORD}';
 		\$CONF['database_name'] = '${POSTFIXADMIN_DB_NAME}';
diff --git a/postfixadmin_latest/Dockerfile b/postfixadmin_latest/Dockerfile
index 6a6628e..6b18dd2 100644
--- a/postfixadmin_latest/Dockerfile
+++ b/postfixadmin_latest/Dockerfile
@@ -1,4 +1,4 @@
-FROM php:7.2-apache
+FROM php:7.3-apache
 LABEL maintainer="David Goodwin <david@codepoets.co.uk> (@DavidGoodwin)"
 
 # Install required PHP extensions
@@ -34,8 +34,8 @@ RUN set -ex; \
 	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
 	rm -rf /var/lib/apt/lists/*
 
-ARG POSTFIXADMIN_VERSION=3.2
-ARG POSTFIXADMIN_SHA512=9e022e3311bce96342e87925159a048f294ed0e2d4558bd43120b5c6eda35a7a2d128987372d310dfe86680c98a4d8cf3d602afcb20d948c9bba58bed1766560
+ARG POSTFIXADMIN_VERSION=3.2.2
+ARG POSTFIXADMIN_SHA512=6c84cb215e69c52c26db0651e5d0d9d8bcb0a63b00d3c197f10fa1f0442a1fde44bb514fb476a1e68a21741d603febac67282961d01270e5969ee13d145121ee
 
 ENV POSTFIXADMIN_VERSION $POSTFIXADMIN_VERSION
 ENV POSTFIXADMIN_SHA512 $POSTFIXADMIN_SHA512
diff --git a/postfixadmin_latest/docker-entrypoint.sh b/postfixadmin_latest/docker-entrypoint.sh
index b8e1f35..b925c5b 100755
--- a/postfixadmin_latest/docker-entrypoint.sh
+++ b/postfixadmin_latest/docker-entrypoint.sh
@@ -5,9 +5,18 @@ POSTFIXADMIN_DB_TYPE=${POSTFIXADMIN_DB_TYPE:=sqlite}
 POSTFIXADMIN_DB_HOST=${POSTFIXADMIN_DB_HOST:=""}
 POSTFIXADMIN_DB_USER=${POSTFIXADMIN_DB_USER:=""}
 POSTFIXADMIN_DB_PASSWORD=${POSTFIXADMIN_DB_PASSWORD:=""}
-POSTFIXADMIN_SETUP_PASSWORD=${POSTFIXADMIN_SETUP_PASSWORD:=""}
+
+# topsecret99
+DEFAULT_SETUP_PASSWORD="791eb4ead7fd996c01bed30707ae27dd:b7910d09773104bf84c4f4951205d2198c7cfc4f"
+POSTFIXADMIN_SETUP_PASSWORD=${POSTFIXADMIN_SETUP_PASSWORD:=$DEFAULT_SETUP_PASSWORD}
+
 
 if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
+
+	if [ "${POSTFIXADMIN_SETUP_PASSWORD}" = "${DEFAULT_SETUP_PASSWORD}" ]; then
+		echo >&2 "WARNING: Using default setup.php password of : topsecret99"
+	fi
+
 	if ! [ -e index.php -a -e scripts/postfixadmin-cli.php ]; then
 		echo >&2 "Postfixadmin not found in $PWD - copying now..."
 		if [ "$(ls -A)" ]; then
@@ -58,6 +67,7 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 		echo "<?php
 		\$CONF['database_type'] = '${POSTFIXADMIN_DB_TYPE}';
 		\$CONF['database_host'] = '${POSTFIXADMIN_DB_HOST}';
+		\$CONF['database_port'] = '${POSTFIXADMIN_DB_PORT}';
 		\$CONF['database_user'] = '${POSTFIXADMIN_DB_USER}';
 		\$CONF['database_password'] = '${POSTFIXADMIN_DB_PASSWORD}';
 		\$CONF['database_name'] = '${POSTFIXADMIN_DB_NAME}';
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #5555; c9d33dbd; `amd64` (`postfixadmin`):

```console

$ bashbrew build postfixadmin:3.2.2-apache
Using bashbrew/cache:8c01369626d598d95d1eaf9b114b5b866e32d17796810e19c9add5e6a29fde4f (postfixadmin:3.2.2-apache)
Tagging postfixadmin:3.2.2-apache
Tagging postfixadmin:3.2-apache
Tagging postfixadmin:3-apache
Tagging postfixadmin:apache
Tagging postfixadmin:3.2.2
Tagging postfixadmin:3.2
Tagging postfixadmin:3
Tagging postfixadmin:latest

$ test/run.sh postfixadmin:3.2.2-apache
testing postfixadmin:3.2.2-apache
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build postfixadmin:3.2.2-fpm
Using bashbrew/cache:e4f2f5c79eff79904a3e89dbfa7fdda6823e9f6fce6c60e0b381f03c72426392 (postfixadmin:3.2.2-fpm)
Tagging postfixadmin:3.2.2-fpm
Tagging postfixadmin:3.2-fpm
Tagging postfixadmin:3-fpm
Tagging postfixadmin:fpm

$ test/run.sh postfixadmin:3.2.2-fpm
testing postfixadmin:3.2.2-fpm
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build postfixadmin:3.2.2-fpm-alpine
Using bashbrew/cache:be67b478782b532ed36b4347732bd5b5fcb44e8b239416e014f4fb45c5ca1d41 (postfixadmin:3.2.2-fpm-alpine)
Tagging postfixadmin:3.2.2-fpm-alpine
Tagging postfixadmin:3.2-fpm-alpine
Tagging postfixadmin:3-fpm-alpine
Tagging postfixadmin:fpm-alpine

$ test/run.sh postfixadmin:3.2.2-fpm-alpine
testing postfixadmin:3.2.2-fpm-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Comment by tianon** [comment]
Looks like this could use an update to 3.2.3. :smile:

**Pull request merged**